packages/wdio-browserstack-service/src/exitHandler.ts

```diff
@@ -9,10 +10,26 @@ import { BStackLogger } from './bstackLogger.js'
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = path.dirname(__filename)
 
+function getSignalName(code: number) : string|null {
```

Comment:
`getSignalName(code: number|indefined)`
since we are accepting code as undefined also

Reply:
using nullable which will handle this